debian/confluent-ksqldb.postinst

```diff
@@ -10,25 +10,28 @@ case "$1" in
 	(getent group $_group 2>&1 >/dev/null) || addgroup --quiet --system $_group
         adduser --system --disabled-password --disabled-login --home /var/empty \
                 --no-create-home --quiet --force-badname --ingroup $_group $_user
+                
+  # Set explicit kafka-streams directory if not set
+  ksqldb_config=/etc/ksqldb/ksql-server.properties
+  if [ -f $ksqldb_config ]; then
+    if ! grep '^state.dir=' $ksqldb_config >/dev/null 2>&1; then
+      echo "state.dir=/var/lib/kafka-streams" >> $ksqldb_config
+    fi
+  fi                
 
 	_permwarn=
-	for dir in /var/log/confluent ; do
-	    if [ $dir = /var/log/confluent ]; then
-		# Confluent log directory should be writable by group
+	for dir in /var/log/confluent /var/lib/kafka-streams ; do
+		# Confluent log and kafka-streams directories should be writable by group
 		_perm="chown ${_user}:${_group} $dir && chmod u+wx,g+wx,o= $dir"
-	    else
-		# Other dirs are only readable by group
-		_perm="chown ${_user}:${_group} $dir && chmod u+wx,g+r,o= $dir"
```

Comment:
any reason why we removed this?

Reply:
It's not used. It was supposed to set a read-only permission on groups for directories other than `/var/log/confluent`. But that's the only directory it was used, so this line was never triggered.

This was the code before:
```
for dir in /var/log/confluent ; do
  if [ $dir = /var/log/confluent ]; then
    // read/write perms
  else
    // read-only perms
  fi
```